vmod/vmod_std.vcc

```diff
@@ -576,7 +576,7 @@ Example::
 		std.set_ip_tos(0);
 	}
 
-$Restrict client
```

Comment:
This is `$Restrict` for `set_ip_tos`, `$Restrict` always apply to the [**preceding**](https://varnish-cache.org/docs/trunk/reference/vmod.html#functions-and-methods-scope-restriction) function/method

Reply:
Ah shoot, I'll fix, thanks